packages/oid4vci-demo-frontend/src/configs/banqup-identity.json

```diff
@@ -0,0 +1,249 @@
+{
+  "general": {
+    "credentialName": "Identity credential",
+    "issueCredentialType": "VerifiedID",
+    "oid4vpAgentBaseUrl": "https://identity.banqup.demo.sphereon.com/agent",
+    "oid4vciAgentBaseUrl": "https://identity.banqup.demo.sphereon.com/agent",
+    "verifierUrl": "https://sphereon.com",
+    "verifierUrlCaption": "Identity credential",
+    "downloadUrl": "https://bank.demo.sphereon.com/agent/#/download",
+    "redirectUrl": "https://knb.demo.sphereon.com"
```

Comment:
i think this redirectUrl is old config and casn go, we do need to make sure that the flow redirects back to the landing page. check with @sanderPostma for that

Reply:
I have in my deployment branch
```
  "general": {
    "credentialName": "Identity credential",
    "oid4vpAgentBaseUrl": "https://identity.banqup.demo.sphereon.com/agent",
    "oid4vciAgentBaseUrl": "https://identity.banqup.demo.sphereon.com/agent",
    "downloadUrl": "https://agent.banqup.demo.sphereon.com/#/download",
    "redirectUrl": "https://banqup.demo.sphereon.com",
    "verifierUrl": "https://sphereon.com",
    "verifierUrlCaption": "Identity credential"
  },
```
(No clue what the last two are)

Reply:
I have this under control now

Reply:
i once added redirectUrl before we had the routes. so this can be removed @sanderPostma @maikel-maas

Reply:
Fixed!